tests/test_unit/test_connections/test_create_all_connection/test_create_clickhouse_connection.py

```diff
@@ -0,0 +1,80 @@
+import pytest
+from httpx import AsyncClient
+from sqlalchemy import select
+from sqlalchemy.ext.asyncio import AsyncSession
+
+from syncmaster.db.models import AuthData, Connection
+from syncmaster.db.repositories.utils import decrypt_auth_data
+from syncmaster.settings import Settings
+from tests.mocks import MockGroup, UserTestRoles
+
+pytestmark = [pytest.mark.asyncio, pytest.mark.backend, pytest.mark.clickhouse]
+
+
+async def test_developer_plus_can_create_clickhouse_connection(
```

Comment:
Shouldn't we also test PATCH with Clickhouse schema?

Reply:
Added, please take a look